### ISSUE TITLE
Are there any special tokens in the answer?

### ISSUE THREAD

**Issue by zd11024** [fudan-zvg/spar#2]
Hi, thanks for your nice work!

I noticed that there are special tokens like <box>[x1, y1, x2, y2]</box> in the answer. I am wondering if there are any other types of special tokens in the answer (like point output or something else). In addition, what learning rate and batch size did you set for LLM during pre-training? Thanks for your assistance!

**Comment by hutchinsonian** [comment]
1. No additional special tokens were added in the vocab. 
2. During pre-training in the paper, the batch size is 256 and the learning rate is 4e-5.

**Comment by zd11024** [comment]
Sorry for the unclear previous description. In some tasks, models are required to predict bounding boxes in formats like "table, <box>[x1, y1, x2, y2]</box>". I want to know whether there are other similar tasks that require special output formats.
